policy-engine/src/graph.rs

```diff
@@ -7,20 +7,27 @@ use cincinnati::CONTENT_TYPE;
 use commons::{self, GraphError};
 use failure::Fallible;
 use futures::{future, Future};
-use prometheus::{Counter, Histogram, HistogramOpts, Registry};
+use prometheus::{Counter, Histogram, histogram_opts, Registry};
 use serde_json;
 use std::collections::HashMap;
 
+pub const GRAPH_SERVE_BUCKETS: &[f64; 11] = &[
```

Comment:
A docstring with a rationale and a short explanation of the values would be nice.

Reply:
Done, PTAL

Reply:
Thanks! I was hoping for a description on the values, but I realize that I simply have to read the prometheus docs for their meaning.